ports/stm32/boards/NUCLEO_F767ZI/mpconfigboard.h

```diff
@@ -12,6 +12,7 @@
 #define MICROPY_HW_ENABLE_RTC       (1)
 #define MICROPY_HW_ENABLE_DAC       (1)
 #define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_ENABLE_SDCARD    (1)
```

Comment:
This board doesn't have a built-in SD card (right?) so I don't think it makes sense to enable it by default.